src/lib/bootloader/grub2_widgets.rb

```diff
@@ -911,6 +930,30 @@ def init_custom_devices(custom_devices)
       end
     end
 
+    # Checks list of custom devices
+    #
+    # @param devs_list[String] comma separated list of device definitions
+    #
+    # @return [Array<String>] devices which didn't pass validation
+    def invalid_custom_devices(devs_list)
+      # almost any byte sequence is potentially valid path in unix like systems
+      # AY profile can be generated for whatever system so we cannot decite if
+      # particular byte sequence is valid or not
+      return [] if Mode.config
+
+      devs_list.split(",").reject do |d|
+        dev_path = DevicePath.new(d)
+
+        if Yast::Mode.installation
+          # uuids are generated later by mkfs, so not known in time of installation
+          # so whatever can be true
```

Comment:
is the statement correct also for labels? or does label have to be specified in advance and known?